### PR TITLE
Add Postgres 17

### PR DESCRIPTION
**Pull request by psavidis**
**Related-to**: https://github.com/camunda/camunda-bpm-platform/issues/4768

### CODE REVIEW

File: stacks/camunda-ci-postgresql-17.0/docker-stack.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+version: '3'
+services:
+  main:
+    deploy:
+      restart_policy:
+        condition: on-failure
+    image: docker.io/library/postgres:17.0
```
Comment (psavidis) [inline stacks/camunda-ci-postgresql-17.0/docker-stack.yml:8]: The Postgres 17 official docker image from docker hub

**Comment by psavidis** [comment]
### Note to Reviewer

The PR is opened using the steps described [here](https://github.com/camunda/portainer-templates?tab=readme-ov-file#how-to-create-a-new-container-type-template).

❓ Is there a way to test it before merging ?

File: stacks/camunda-ci-postgresql-17.0/docker-stack.yml
Comment (yanavasileva) [inline stacks/camunda-ci-postgresql-17.0/docker-stack.yml:1]: ❓ I think camunda db won't be created with this stack. For the other stack we just spin up image and everything is ready to use with default database name, user, password. That's not the case here, right?

**Review by yanavasileva — COMMENTED** [review]
Let's discuss how the portainer will be used. See my question below.
The rest looks good.

**Comment by yanavasileva** [comment]
💭 Do we need to execute this script to create the schema?
https://github.com/camunda/cambpm-jenkins-shared-library/blob/master/vars/cambpmConditionalRetry.groovy#L600-L623
Reference for mssql init script: [here](https://github.com/camunda/camunda-docker-ci-mssql/blob/master/2022/script/init.sh).
pg 16: [here](https://github.com/camunda/camunda-docker-ci-postgresql/tree/master/16.0)

File: stacks/camunda-ci-postgresql-17.0/docker-stack.yml
Comment (psavidis) [reply stacks/camunda-ci-postgresql-17.0/docker-stack.yml:1]: Great point. Indeed while testing `Postgres 17` with a docker image, the engine requires an existing db in the server, which is not created unless the following env variables are specified on startup:

- `POSTGRES_USER`
- `POSTGRES_PASSWORD`
- `POSTGRES_DB`

I've pushed a [commit](https://github.com/camunda/portainer-templates/pull/70/commits/f5b106611da1534d4568af90e04cfb6b879fe48b) which adds the default hardcoded values.

❓ Should the population of these values be externalised to other variables?

In other servers e.g `weblogic14r1-720-latest`, the env variables receive values from other ones ([ref](https://github.com/camunda/portainer-templates/blob/45da769e4d0d1f44abd0a79f891ac176879eb98c/stacks/weblogic14r1-720-latest/docker-stack.yml#L9-L12)).

I can find the variable definitions ([ref](https://github.com/camunda/portainer-templates/blob/45da769e4d0d1f44abd0a79f891ac176879eb98c/stack-templates.json#L446-L463)) but not the respective values 🤔

**Review by tasso94 — APPROVED** [review]
LGTM!

**Pull request merged**